widgets/sites/src/SiteActions.ts

```diff
@@ -1,23 +1,32 @@
-// @ts-nocheck File not migrated fully to TS
+import type { Visibility } from 'app/widgets/common/types';
+import { isNil, omitBy } from 'lodash';
+
 export default class SiteActions {
-    constructor(toolbox) {
+    toolbox: Stage.Types.WidgetlessToolbox;
+
+    constructor(toolbox: Stage.Types.WidgetlessToolbox) {
         this.toolbox = toolbox;
     }
 
-    doGet(name) {
+    doGet(name: string) {
         return this.toolbox.getManager().doGet(`/sites/${name}`);
```

Comment:
With something like (haven't tested, don't know exact response)
```suggestion
        return this.toolbox.getManager().doGet<Site>(`/sites/${name}`);
```
you'll be able to easier type places which uses this method.

Reply:
As I'm investigating right now, this method is not being used, so I'll just remove it 😁